### ISSUE TITLE
Feature: redirect `/defaultLocale/about` to `/about`

### ISSUE THREAD

**Issue by jlarmstrongiv** [jlarmstrongiv/astro-i18n-aut#2]
Depends on https://github.com/withastro/astro/issues/7322

**Comment by jlarmstrongiv** [comment]
Hey @lemueldls since I have this project open, I’m going to go ahead and implement this feature.  Would you test it on windows afterwards for me?

**Comment by lemueldls** [comment]
Yeah, sure thing!

**Comment by jlarmstrongiv** [comment]
@lemueldls it turns out that fixing windows paths broke mac paths, so I changed the logic in `0.0.22`.  

Apparently `fg.convertPathToPattern` was actually escaping the globs, not just the file paths.

I tried to ensure that every path was normalized and removed windows `\\` when needed.  

Let me know if it builds on your machine!  It currently builds on Mac.  You will either need to add middleware (automatically generated for you), or set `redirectDefaultLocale` to false in this update.
